### PR TITLE
Awso280 tf

### PR DESCRIPTION
**Pull request by himsharma01**

### CODE REVIEW

File: aws-observability-terraform/README.md [hunk 1]
```diff
@@ -5,15 +5,15 @@
 | <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.13.0 |
 | <a name="requirement_aws"></a> [aws](#requirement\_aws) | >= 5.16.2, < 6.0.0 |
 | <a name="requirement_random"></a> [random](#requirement\_random) | >= 3.1.0 |
-| <a name="requirement_sumologic"></a> [sumologic](#requirement\_sumologic) | >= 2.14.0 |
+| <a name="requirement_sumologic"></a> [sumologic](#requirement\_sumologic) | >= 2.28.3, < 3.0.0 |
 | <a name="requirement_time"></a> [time](#requirement\_time) | >= 0.7.1 |
```
Comment (npande) [inline aws-observability-terraform/README.md:9]: all references to time should be consistent - move from 0.7.1 to 0.11.1. I see multiple instances missing the same, possibly this needs to be updated in the tf code and readme tool will pick up from there

Comment (himsharma01) [reply aws-observability-terraform/README.md:9]: Sure let me update

File: aws-observability-terraform/test/appmodule/validateSumo.go [hunk 2]
```diff
@@ -204,8 +207,14 @@ func validateSumoLogicResources(t *testing.T, workingDir string) {
 	dbinstanceidentifierField := terraform.Output(t, terraformOptions, "sumologic_field_dbinstanceidentifier")
 	validateSumoLogicField(t, terraformOptions, dbinstanceidentifierField)
 	time.Sleep(2 * time.Second)
+	// Validate if the dbinstanceidentifier Field is created successfully
```
Comment (himanshu219) [inline aws-observability-terraform/test/appmodule/validateSumo.go:210]: we can remove the comment and may be just have generic `Validate field creation` comment

Comment (himsharma01) [reply aws-observability-terraform/test/appmodule/validateSumo.go]: Thanks @himanshu219 for pointing it out. Updated the comment.

**Pull request merged**